site/docs/plugins/commands.md

```diff
@@ -5,10 +5,628 @@ next: false
 
 # Commands (`commands`)
 
-Coming soon, please come back later.
+Command handling on steroids.
+
+This plugin provides various features related to command handling that are not contained in the [command handling done by the core library](../guide/commands).
+Here is a quick overview of what you get with this plugin:
+
+- Better code readability by encapsulating middleware with command definitions
+- User command menu synchronization via `setMyCommands`
+- Improved command grouping and organization
+- Ability to scope command reach, e.g: only accessible to group admins or
+  channels, etc
+- Defining command translations
+- `Did you mean ...?` feature that finds the nearest existing command to a given
+  user miss-input
+- Case-insensitive command matching
+- Setting custom behavior for commands that explicitly mention your bot's user,
+  like: `/start@your_bot`
+- Custom command prefixes, e.g: `+`, `?` or any symbol instead of `/`
+- Support for commands that are not in the beginning of the message
+- RegExp Commands!
+
+All of these features are made possible because you will define one or more central command structures that define your bot's commands.
+
+## Basic Usage
+
+Before we dive in, take a look at how you can register and handle a command with the plugin:
+
+```ts
+const myCommands = new CommandGroup();
+
+myCommands.command("hello", "Say hello", (ctx) => ctx.reply(`Hello, world!`));
+
+bot.use(myCommands);
+```
+
+This registers a new `/start` command to your bot that will be handled by the given middleware.
```

Comment:
Aren't should be:

```ts
myCommands.command("hello", "Say hello", (ctx) => ctx.reply(`Hello, world!`));

bot.use(myCommands);
```

This registers a new `/hello` command to your bot that will be handled by the given middleware.

Reply:
`/start` should be changed to `/hello` I mean

Reply:
```suggestion
myCommands.command("hello", "Say hello", (ctx) => ctx.reply(`Hello, world!`));

bot.use(myCommands);
```

This registers a new `/hello` command to your bot that will be handled by the given middleware.
```

Reply:
couldn't commit it from the UI for some reason